### DISCUSSION TITLE
warning from Ansible failing to find templates when run by ansible-lint

### DISCUSSION THREAD

**Discussion by iranzo** [ansible/ansible-lint#1386] [Q&A]
##### Summary

With previous version of ansible-lint, templates were found in the 'templates' folder for  a role, and it seems, according to https://github.com/ansible/ansible/issues/17960#issue-181994152 that t he 'templates' folder is the one to use

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible 2.9.17
  config file = /home/iranzo/.ansible.cfg
  configured module search path = ['/home/iranzo/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.9/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.9.1 (default, Jan 20 2021, 00:00:00) [GCC 10.2.1 20201125 (Red Hat 10.2.1-9)]


ansible-lint 4.3.5

```

- ansible installation method: one of source, pip, OS package
- ansible-lint installation method: one of source, pip, OS package

ansible-2.9.17-2.fc33.noarch
python3-ansible-lint-4.3.5-1.fc33.noarch

##### OS / ENVIRONMENT

Fedora 33

##### STEPS TO REPRODUCE

```console (paste below)

```

![image](https://user-images.githubusercontent.com/312463/108479023-7eebac80-7295-11eb-992f-185e49074ef3.png)

~~~sh
find|grep dir.xml
./ansible-ipi-install/roles/node-prep/templates/dir.xml.j2
~~~

##### Desired Behaviour

It should have found the templates as it did in the past

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behaviour

It doesn't find the tempaltes

Please give some details of what is actually happening.
Include a [minimum complete verifiable example] with:
- playbook

Playbook at https://github.com/openshift-kni/baremetal-deploy/tree/master/ansible-ipi-install

- output of running ansible-lint
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`

```paste below

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by ssbarnea** [comment]
I converted this to a discussion as this report did not follow multiple sections ofthe guidelines: the example is not minimal, in fact the mentioned repository reports hundreads of unrelated violations, making very hard to dig on whatever is happening here.

The reality is that there is a bug as I seen ansible syntax check not creating these warnings, but without a really minimized case to reproduce it I doubt anyone would work on fixing it. Most bugs can be reproduced with a single file, but in some cases you can create a temporary git repository that contains minimal code to produce this issue and nothing else.

I also observed that you did not use the latest version of the linter, which is required for any report. In that case it did not matter as I was able to reproduce it but is not ok to expect maintainers to do this.
